azurerm_lb_backend_address_pool: support for `backend_address` (#10291)
terraform-providers/terraform-provider-azurerm: website/docs/r/lb_backend_address_pool.html.markdown

The following arguments are supported:

* `name` - (Required) Specifies the name of the Backend Address Pool.
  

> * `resource_group_name` - (Optional / **Deprecated**) The name of the resource group in which to create the resource.

Reviewer: We can just remove deprecated properties from the docs
```suggestion
```

  
* `loadbalancer_id` - (Required) The ID of the Load Balancer in which to create the Backend Address Pool.

* `backend_address` - (Optional) An array of `backend_address` block as defined below.

> -> **NOTE**: The `backend_address` is only allowed when the Load Balancer in which this Backend Address Pool resides is of `Standard` sku. See [the Azure document](https://docs.microsoft.com/en-us/azure/load-balancer/backend-pool-management#limitations) for more details.

Reviewer: grammer
```suggestion
-> **NOTE**: The `backend_address` can only be configured when the Backend Address Pool Load Balancer's SKU is `Standard`. See [the Azure document](https://docs.microsoft.com/en-us/azure/load-balancer/backend-pool-management#limitations) for more details.
```
